slices/openjdk-21-jre-headless.yaml

```diff
@@ -0,0 +1,197 @@
+package: openjdk-21-jre-headless
+
+# OpenJDK slices provide required dependencies for the runtime image.
+# The image itself is built using jlink.
+# The slices can be used to create build image.
```

Comment:
This slice is for the JRE, not a build image. am i missing something?

Reply:
I wanted to say that using slice will result in bigger image than jlink. The jre slice is necessary for jdk slice and the latter can be used to run jlink to create an optimized image.

I have droppped comment.